api/lib/mno_enterprise/concerns/controllers/webhook/o_auth_controller.rb

```diff
@@ -57,17 +62,19 @@ def error_message(error_key)
   #==================================================================
   # GET /mnoe/webhook/oauth/:id/authorize
   def authorize
-    if params[:redirect_path].present?
-      session[:redirect_path] = params[:redirect_path]
-    end
-
     # Certain providers require options to be selected
     if !params[:perform] && app_instance.app && PROVIDERS_WITH_OPTIONS.include?(app_instance.app.nid.to_s)
       render "mno_enterprise/webhook/o_auth/providers/#{app_instance.app.nid}"
       return
     end
 
+    # MnoHub redirection to perform the authorise call
     @redirect_to = MnoEnterprise.router.authorize_oauth_url(params[:id], extra_params.merge(wtk: wtk))
+
+    if ENV['SKIP_REDIRECTION_SPEEDBUMPS']
```

Comment:
Should we use a [feature flag](https://github.com/maestrano/mno-enterprise/blob/5.0/docs/feature-flags.md) instead?

If the Tenant is not allowed to change this, we can have it as `Tenant#metadata` flag